Tools/environment_install/install-prereqs-arch.sh

```diff
@@ -33,6 +33,7 @@ PYTHON_PKGS="future lxml pymavlink MAVProxy pexpect argparse matplotlib pyparsin
 ARM_ROOT="gcc-arm-none-eabi-10-2020-q4-major"
 ARM_TARBALL="$ARM_ROOT-x86_64-linux.tar.bz2"
 ARM_TARBALL_URL="https://firmware.ardupilot.org/Tools/STM32-tools/$ARM_TARBALL"
+ARM_TARBALL_CHECKSUM="21134caa478bbf5352e239fbc6e2da3038f8d2207e089efc96c3b55f1edcd618" 
```

Comment:
@peterbarker that is totally something we will forget to update. Can't we add a checksum file on the server instead ?

Reply:
@khancyr I really did consider doing that - including getting a shell on the server to create the file.

Then I thought again.  We will *not* be updating any package on the server to have a different checksum - we never have, and I can't see why we would.  Everything's a mirror of something someone else has done, and we're not stripping version information from the file we're downloading or anything along those lines.

I decided on balance that dropping the expected checksum in here was fine, and that I didn't need to complicate things!

Reply:
... I also considered how often we update our release compiler and decided it was unlikely it would be a massive problem.